subprojects/docs/src/docs/userguide/distributionPlugin.adoc

```diff
@@ -18,37 +18,37 @@
 
```

Comment:
❓ For the title, could you please remove "The"?

Reply:
I decided against that for now because the other chapters I looked at (e.g. Java Plugin, Signing Plugin) also use "The".